content/en/docs/tasks/observability/distributed-tracing/zipkin/index.md

```diff
@@ -41,12 +41,11 @@ $ istioctl dashboard zipkin
 
     {{< boilerplate trace-generation >}}
 
-1.  From the top panel, select a service of interest (or 'all') from the **Service Name** drop-down list and click
-    **Find Traces**:
+1.  From the search panel, click on the plus sign. Select `serviceName` from the first dropdown and `productpage.default` from second dropdown and click the search icon:
 
     {{< image link="./istio-tracing-list-zipkin.png" caption="Tracing Dashboard" >}}
 
-1.  Click on the most recent trace at the top to see the details corresponding to the
+1.  Click on the `TRACE ID` to see the details corresponding to the
```

Comment:
Same comment about correlation with the following diagram.

Reply:
Click on the `ISTIO-INGRESSGATEWAY` from the above list to see the details corresponding to the latest request to `/productpage`

Does this sound good?

Reply:
Sounds good with minor tweak:
```suggestion
1.  Click on the `ISTIO-INGRESSGATEWAY` search result to see the details corresponding to the
```

Reply:
Done. Thanks.